python/cudf/cudf/_lib/table.pyx

```diff
@@ -0,0 +1,74 @@
+from cudf._lib.cudf cimport *
+from libcpp.memory cimport unique_ptr, make_unique
+
+cdef class TableView:
+    """
+    A non-owning view into a set of Columns
+    """
+    def __cinit__(self, columns):
+        cdef gdf_column* c_col
+        for col in columns:
+            c_col = column_view_from_column(col)
+            self.c_columns.push_back(c_col)
+        self.ptr = new cudf_table(self.c_columns)
+        self.own_ptr = True
```

Comment:
Shouldn't this be false? You don't want to free the underlying columns by default, no?

Reply:
100%. I forgot to remove `own_ptr` from `TableView`.

Reply:
Done